src/modules/exposure-period/exposure-period.service.ts

```diff
@@ -19,7 +19,7 @@ export class ExposurePeriodService {
       // TODO: SP USP_MDM_READ_EXPOSURE_PERIOD is not using data/tables from DB. Calculation could be moved to Javascript.
       const results = await this.mdmDataSource.query('USP_MDM_READ_EXPOSURE_PERIOD @0, @1, @2', [startDate, endDate, productGroup]);
 
-      if (!results.Length || typeof results[0].EXPOSURE_PERIOD === 'undefined') {
+      if (!results.length || typeof results[0].EXPOSURE_PERIOD === 'undefined') {
```

Comment:
Do we need index `0` void check before accessing it?

Reply:
`results` will always be an array - so the length will either be 0 or more

Reply:
Marking as resolve, we used to use index 0, but don't do it anymore.